Source/WebCore/platform/mediastream/RealtimeMediaSourceCenter.cpp

```diff
@@ -157,12 +157,12 @@ static void addStringToSHA1(SHA1& sha1, const String& string)
     if (string.is8Bit() && string.containsOnlyASCII()) {
         const uint8_t nullByte = 0;
         sha1.addBytes(string.span8());
-        sha1.addBytes(&nullByte, 1);
+        sha1.addBytes(std::span { &nullByte, 1 });
         return;
     }
 
     auto utf8 = string.utf8();
-    sha1.addBytes(utf8.dataAsUInt8Ptr(), utf8.length() + 1); // Include terminating null byte.
+    sha1.addBytes(std::span { utf8.dataAsUInt8Ptr(), utf8.length() + 1 }); // Include terminating null byte.
```

Comment:
`sha1.addBytes(string.utf8().spanIncludingNullTerminator());`

No need for the `utf8` local anymore either.

Reply:
Nice!